src/components/RegionItem/RegionItem.style.tsx

```diff
@@ -31,22 +32,52 @@ export const StyledLink = styled(Link)`
   text-decoration: none;
 `;
 
-export const Wrapper = styled.div`
+export const SharedWrapperStyles = css`
   display: flex;
   align-items: center;
   background-color: ${COLOR_MAP.LIGHTGRAY_BG};
   max-width: 330px;
   width: 100%;
   padding: 1.25rem 1.75rem 1.25rem 1.25rem;
-  cursor: pointer;
   border-radius: 4px;
+`;
+
+export const Wrapper = styled.div`
+  ${SharedWrapperStyles};
+  cursor: pointer;
 
   :hover {
-    background-color: ${COLOR_MAP.BLUE};
-    color: white;
+    box-shadow: 0px 2px 24px rgba(0, 0, 0, 0.12);
+  }
+`;
+
+/* For loading state: */
+
+export const SkeletonWrapper = styled.div`
+  ${SharedWrapperStyles};
+  height: 84px;
 
-    ${ArrowIcon},${LevelDescription} {
-      color: white;
-    }
+  ${ArrowIcon} {
+    color: ${COLOR_MAP.GRAY_EXPLORE_CHART};
+  }
+`;
+
+export const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.25rem;
+`;
+
+export const StyledSkeletonRect = styled(Skeleton).attrs(props => ({
+  height: 14,
+  animation: 'wave',
+}))`
+  border-radius: 0;
+
+  &:first-of-type {
+    width: 210px;
+  }
+  &:last-of-type {
+    width: 75px;
```

Comment:
Consider adding props for `width` and `height` to make it more reusable, that way we can put more than 2 and make more complex constructs

```
<StyledSkeletonRect height={15} width={80} /> // title
<StyledSkeletonRect height={80} width={140} /> // image
<StyledSkeletonRect height={12} width={140} /> // text...
<StyledSkeletonRect height={12} width={140} />
<StyledSkeletonRect height={12} width={140} />
```